surfer/viz.py

```diff
@@ -492,6 +510,77 @@ def _set_window_properties(self, size, background, foreground):
         fg_color_rgb = colorConverter.to_rgb(foreground)
         self._fg_color = fg_color_rgb
 
+    def _get_geo_params(self, cortex, alpha=1.0):
+        """Return keyword arguments and other parameters for surface
+        rendering.
+
+        Parameters
+        ----------
+        cortex : {classic, high_contrast, low_contrast, bone, tuple}
+            The name of one of the preset cortex styles, or a tuple
+            with four entries as described in the return vales.
```

Comment:
Looking at the return values documentation, it's not clear what the tuple entries should be. (There are three return values for this function).